### ISSUE TITLE
Cannot read property '0' of undefined

### ISSUE THREAD

**Issue by snowcxt** [Glavin001/atom-beautify#646]
Used 0.28.17 to format a jsx file. The Stack Trace

```
TypeError: Cannot read property '0' of undefined
  at jspretty__algorithm_start (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\jspretty.js:2516:48)
  at jspretty__algorithm (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\jspretty.js:3298:21)
  at jspretty_ (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\jspretty.js:3329:10)
  at commonjs (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\jspretty.js:4958:16)
  at markuppretty__apply_jsxattribute (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\markuppretty.js:1752:46)
  at markuppretty__apply (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\markuppretty.js:1762:17)
  at markuppretty_ (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\markuppretty.js:2573:6)
  at commonjs (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\markuppretty.js:2606:16)
  at jspretty__result_markupwrapper (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\jspretty.js:4351:43)
  at jspretty__result (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\jspretty.js:4530:33)
  at jspretty_ (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\jspretty.js:4603:14)
  at commonjs (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\jspretty.js:4958:16)
  at markuppretty__beautify_script (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\markuppretty.js:1398:39)
  at markuppretty__beautify (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\markuppretty.js:1526:21)
  at markuppretty_ (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\markuppretty.js:1545:10)
  at commonjs (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\markuppretty.js:2606:16)
  at jspretty__result_markupwrapper (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\jspretty.js:4351:43)
  at jspretty__result (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\jspretty.js:4530:33)
  at jspretty_ (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\jspretty.js:4603:14)
  at commonjs (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\lib\jspretty.js:4958:16)
  at core_ (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\prettydiff.js:972:41)
  at prettydiff_ (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\prettydiff.js:1643:12)
  at Object.commonjs [as api] (C:\Users\tcui\.atom\packages\atom-beautify\node_modules\prettydiff\prettydiff.js:1678:16)
  at C:\Users\tcui\.atom\packages\atom-beautify\src\beautifiers\prettydiff.coffee:110:27
  at tryCatcher (C:\Users\tcui\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\babel-core\node_modules\bluebird\js\main\util.js:26:23)
  at Promise._resolveFromResolver (C:\Users\tcui\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\babel-core\node_modules\bluebird\js\main\promise.js:480:31)
  at new Promise (C:\Users\tcui\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\babel-core\node_modules\bluebird\js\main\promise.js:70:37)
  at PrettyDiff.module.exports.PrettyDiff.beautify (C:\Users\tcui\.atom\packages\atom-beautify\src\beautifiers\prettydiff.coffee:59:16)
  at C:\Users\tcui\.atom\packages\atom-beautify\src\beautifiers\index.coffee:455:24
  at tryCatcher (C:\Users\tcui\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\babel-core\node_modules\bluebird\js\main\util.js:26:23)
  at Promise._resolveFromResolver (C:\Users\tcui\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\babel-core\node_modules\bluebird\js\main\promise.js:480:31)
  at new Promise (C:\Users\tcui\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\babel-core\node_modules\bluebird\js\main\promise.js:70:37)
  at C:\Users\tcui\.atom\packages\atom-beautify\src\beautifiers\index.coffee:325:18
  at tryCatcher (C:\Users\tcui\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\babel-core\node_modules\bluebird\js\main\util.js:26:23)
  at Promise._settlePromiseFromHandler (C:\Users\tcui\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\babel-core\node_modules\bluebird\js\main\promise.js:507:31)
  at Promise._settlePromiseAt (C:\Users\tcui\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\babel-core\node_modules\bluebird\js\main\promise.js:581:18)
  at Promise._settlePromises (C:\Users\tcui\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\babel-core\node_modules\bluebird\js\main\promise.js:697:14)
  at Async._drainQueue (C:\Users\tcui\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\babel-core\node_modules\bluebird\js\main\async.js:123:16)
  at Async._drainQueues (C:\Users\tcui\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\babel-core\node_modules\bluebird\js\main\async.js:133:10)
  at MutationObserver.Async.drainQueues (C:\Users\tcui\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\babel-core\node_modules\bluebird\js\main\async.js:15:14)
```

**Comment by prettydiff** [comment]
I am looking into this.

**Comment by prettydiff** [comment]
Resolved in published version v1.15.14.
